llvm/lib/Target/RISCV/RISCVInstrInfoXqci.td

```diff
@@ -1186,6 +1186,23 @@ def PseudoLongQC_E_BGEI : LongBcciPseudo<simm16nonzero, 10>;
 def PseudoLongQC_E_BLTUI : LongBcciPseudo<uimm16nonzero, 10>;
 def PseudoLongQC_E_BGEUI : LongBcciPseudo<uimm16nonzero, 10>;
 
+// Load/Store predicates with QC.E.* Mnemonics. These expand to an AUIPC +
+// (Standard) Load/Store pairs, as this sequence can materialize all 32-bit
```

Comment:
Do we need the word "pair" in these sentences? I only ask because "load/store pair" is the name of the Zilsd extension. So I briefly parsed the sentence wrong.